js/components/closed-caption-multi-audio-menu/tab.js

```diff
@@ -19,24 +22,36 @@ var Tab = React.createClass({
           speed={1}
           horizontal={false}
         >
-          <ul className="oo-cc-ma-menu__list">
+          <ul
+            className="oo-cc-ma-menu__list"
+            role={CONSTANTS.ARIA_ROLES.MENU}
+          >
             {this.props.itemsList.map(function(el, index) {
               return (
                 <li
                   key={index}
-                  onClick={this.handleClick.bind(this, el.id)}
                   className={classnames('oo-cc-ma-menu__element', {
                     'oo-cc-ma-menu__element--active': el.enabled
                   })}
                 >
-                  <Icon
-                    skinConfig={this.props.skinConfig}
-                    icon="selected"
-                    className={classnames({ 'oo-icon-hidden': !el.enabled })}
-                  />
-                  <span className="oo-cc-ma-menu__name" title={el.label}>
+                  <AccessibleButton
+                    key={index}
+                    className={'oo-ma-btn'}
+                    focusId={CONSTANTS.FOCUS_IDS.MULTI_AUDIO + index}
+                    role={CONSTANTS.ARIA_ROLES.MENU_ITEM_RADIO}
+                    ariaLabel={el.label}
+                    ariaChecked={el.enabled}
+                    onClick={this.handleClick.bind(this, el.id)}
+                  >
+                    <Icon
+                      skinConfig={this.props.skinConfig}
+                      icon="selected"
+                      className={classnames({ 'oo-icon-hidden': !el.enabled })}
+                    />
+                    <span className="oo-cc-ma-menu__name" title={el.label}>
```

Comment:
Do we need the title attribute?

Reply:
The attribute is used in styles to prevent jitter of lines when hovering (according to the design, the font becomes bold when hovering).